src/main/java/com/lob/exception/APIException.java

```diff
@@ -7,6 +7,8 @@
 
 public class APIException extends LobException {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
Is there any detriment in letting the compiler generate these. I'm curious if they are being used in implementations somewhere

Reply:
IDE dropped them in. 

There shouldn't be any issue with them being there but let me rip them out for consistency.